httplib.h

```diff
@@ -1667,14 +1667,16 @@ inline bool compress(std::string &content) {
 class decompressor {
 public:
   decompressor() {
+    std::memset(&strm, 0, sizeof(strm));
     strm.zalloc = Z_NULL;
     strm.zfree = Z_NULL;
     strm.opaque = Z_NULL;
 
     // 15 is the value of wbits, which should be at the maximum possible value
     // to ensure that any gzip stream can be decoded. The offset of 16 specifies
```

Comment:
Since '32' is now used for the automatic stream type detection, this comment is no longer valid. Could you update it?

Reply:
Yes, for sure